.github/workflows/canary-integration-test.yml

```diff
@@ -0,0 +1,78 @@
+name: Canary integration test
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.13
```

Comment:
Can we use go 1.15?

Reply:
We can, but all our builds are on 1.13 so we better stay consistent, right?